examples/host.hs

```diff
@@ -18,16 +19,18 @@ external systems with which the FRP-enabled program is interacting.
 
 module Main where
 
-import Reflex
-import Reflex.Host.Class (newEventWithTriggerRef, runHostFrame, fireEvents)
-import Control.Concurrent (forkIO)
-import Control.Monad (forever)
-import Control.Monad.Fix (MonadFix)
-import Control.Monad.Identity (Identity(..))
-import Control.Monad.IO.Class (liftIO)
-import Data.IORef (readIORef)
-import Data.Dependent.Sum (DSum ((:=>)))
-import System.IO (hSetEcho, hSetBuffering, stdin, BufferMode (NoBuffering))
+import           Control.Concurrent     (forkIO)
+import           Control.Monad          (forever)
+import           Control.Monad.Fix      (MonadFix)
+import           Control.Monad.Identity (Identity (..))
+import           Control.Monad.IO.Class (liftIO)
+import           Data.Dependent.Sum     (DSum ((:=>)))
+import           Data.IORef             (readIORef)
+import           Reflex
+import           Reflex.Host.Class      (fireEvents, newEventWithTriggerRef,
+                                         runHostFrame)
+import           System.IO              (BufferMode (NoBuffering),
+                                         hSetBuffering, hSetEcho, stdin)
```

Comment:
Yes `stylish-haskell` did this without me realizing it. Perhaps I'm not properly configured to use your settings?